cyral/resource_cyral_integration_idp_saml_draft_test.go

```diff
@@ -122,6 +122,28 @@ func setupIntegrationIdPSAMLDraftTest(t *testing.T, draft *GenericSAMLDraft, res
 		}...)
 	}
 
+	// checking SPMetadata content
+	checkFuncs = append(checkFuncs, []resource.TestCheckFunc{
+		resource.TestMatchResourceAttr(resourceFullName, "service_provider_metadata.0.xml_document",
```

Comment:
If we just want to check if the value was set for the field the [TestCheckResourceAttrSet](https://pkg.go.dev/github.com/hashicorp/terraform-plugin-sdk/helper/resource#TestCheckResourceAttrSet) may be more appropriated ([See](https://developer.hashicorp.com/terraform/plugin/sdkv2/testing/acceptance-tests/teststep#builtin-check-functions))

Reply:
[637f75e](https://github.com/cyralinc/terraform-provider-cyral/pull/394/commits/637f75e408adf92993fa7023e968c975b2bfc972) addr

Reply:
addr [6c98809](https://github.com/cyralinc/terraform-provider-cyral/pull/394/commits/6c9880953be078345e8b73fc3c6013a1e196c8e3)